README.md

```diff
@@ -0,0 +1,79 @@
+# Environment Variables Injector
+
+This extension is designed to help users export snap options configured with the [snap set command](https://snapcraft.io/docs/configuration-in-snaps) into environment variables.
```

Comment:
There are a few ways to set configurations, not just via the `snap set` command.

Reply:
Addressed, thank you!